src/CppUTest/CommandLineArguments.cpp

```diff
@@ -93,9 +94,11 @@ bool CommandLineArguments::parse(TestPlugin* plugin)
 
 const char* CommandLineArguments::usage() const
 {
-    return "use -h for more extensive help\nusage [-h] [-v] [-vv] [-c] [-p] [-lg] [-ln] [-ri] [-r#]\n"
-                                           "      [-g|sg|xg|xsg groupName]... [-n|sn|xn|xsn testName]... [-t groupName.testName]...\n"
-                                           "      [-b] [-s [randomizerSeed>0]] [\"TEST(groupName, testName)\"]... [-o{normal, junit, teamcity}] [-k packageName]\n";
+    return "use -h for more extensive help\n"
+           "usage [-h] [-v] [-vv] [-c] [-p] [-lg] [-ln] [-ri] [-r#] [-f]\n"
+           "      [-g|sg|xg|xsg groupName]... [-n|sn|xn|xsn testName]... [-t groupName.testName]...\n"
+           "      [-b] [-s [randomizerSeed>0]] [\"TEST(groupName, testName)\"]...\n"
+           "      [-o{normal, junit, teamcity}] [-k packageName]\n";
```

Comment:
This last line of this was too long to be reasonably output on a console without wrapping. Hence, split the string whilst adding the new argument.